src/components/Footer/Footer.styles.tsx

```diff
@@ -13,39 +13,85 @@ export const Wrapper = styled.footer`
   gap: 1rem;
   position: relative;
   width: 100%;
-  align-items: center;
+  align-items: start-flex;
 
-  @media (max-width: 600px) {
+  @media (max-width: 800px) {
+    align-items: center;
     flex-direction: column;
   }
 `;
 
 export const Divider = styled.hr`
+  border: none;
+  height: 1px;
   width: 100%;
-  border: 1px solid ${({ theme }) => theme.colors.white};
+  background-color: ${({ theme }) => theme.colors.white};
+  margin-top: 0px;
 `;
 
 export const LogoImage = styled(Icon)``;
 
 export const References = styled.div`
   display: flex;
-
   gap: 0.5rem;
-  align-items: end;
+
   flex-direction: column;
-  justify-content: center;
+  justify-content: flex-end;
 
-  @media (max-width: 600px) {
+  flex-grow: 2;
+
+  @media (max-width: 800px) {
     align-items: center;
+    flex-direction: column;
+    gap: 2rem;
+  }
+`;
+
+export const Sections = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  gap: 2.2rem;
+
+  @media (max-width: 800px) {
+    flex-direction: column;
+  }
+`;
+
+export const SectionTitle = styled.h2`
```

Comment:
```suggestion
export const SectionTitle = styled.h3`
```

I think it would be best if this was a `h3` tag. I'm saying that because the subsections in out frontend are `h2` and they are more important than the footer link. With that, if a person with a screen reader uses our site the subsections and footer section link aren't going to be confused by it